src/components/VDate/datepicker.js

```diff
@@ -623,6 +629,9 @@ DatePicker.prototype.getDateForButtonLabel = function(year, month, day) {
 };
 
 DatePicker.prototype.setMessage = function(str) {
+  /**
+   * [setMessageDelayed description]
```

Comment:
Replace `[setMessageDelayed description]` with a useful description of the variable.